backend/src/Services/MethodCalculationService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Database.GraphQl.MethodAsService;
+using Database.Methods;
+using Database.Services.Interfaces;
+
+namespace Database.Services;
+
+public class MethodCalculationService : IMethodCalculationService
+{
+    private List<IMethod> _methods = new List<IMethod>();
```

Comment:
I would prefer it if for every method ID there could be only one method registered, for example, using a hash map with Guids as keys and `IMethod`s as values.